### ISSUE TITLE
Script all data creations

### ISSUE THREAD

**Issue by vpratfr** [bordoni/fakerpress#55]
Would be great to be able to script the data creation. This could be used to create a new website instance from scratch.

Could be for instance: 
- I create my data (users for instance)
- if I am happy with the settings I used, I can save them as a JSON file for example
- Then I create other kind of data (posts, terms, ...)

Once that is done, I have a bunch of JSON files. I can use the Reset WordPress plugin to get back to an empty site. 

You can then have a FakerPress page where I could simply import all my JSON files and let the plugin create all the data for me.

Another way to do that, would be to record the creation steps parameters as a setting.

**Comment by bordoni** [comment]
Hi @vpratfr, I'm still thinking on how to implement this feature, I still think there is a few improvements that should come first. So I will say that this will be done, but not on version `0.4.0`, maybe on `0.5.*` or `0.6.*`.

There are 2 things I think the plugin is missing for this to happen, the AJAX on #53 and a way to generate settings for a WordPress Site.

**Comment by vpratfr** [comment]
I agree it totally depends on #53 and not a trivial feature to implement. Thanks a lot, your plugin is very helpful, I had been thinking about that myself for quite some time. Glad you did it and also glad you did it well.

**Comment by bordoni** [comment]
Thanks man! Lets see how thing evolve after #53

**Comment by bordoni** [comment]
This will be resolved later on by using `WP-CLI`, duplicate of #88
